packages/riipen-ui/src/components/Avatar.test.jsx

```diff
@@ -0,0 +1,167 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Avatar from "./Avatar";
+
+describe("<Avatar>", () => {
+  it("renders without errors", () => {
+    let error;
+    try {
+      mount(<Avatar />);
+    } catch (e) {
+      error = e;
+    }
+    expect(error).toEqual(undefined);
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const wrapper = mount(<Avatar />);
+
+      expect(wrapper.find("Avatar").props().size).toEqual("96px");
+      expect(wrapper.find("Avatar").props().variant).toEqual("circle");
+
+      expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    it("appends higher order values to default classes prop with withClass decorator", () => {
+      const wrapper = mount(<Avatar />);
+
+      expect(wrapper.find("Avatar").props().classes).toEqual([
+        "riipen",
+        "riipen-avatar"
+      ]);
+    });
+  });
+
+  describe("alt prop", () => {
+    it("sets inner class name given custom alt props", () => {
+      const altVariant = "Test String";
+
+      const wrapper = mount(<Avatar alt={altVariant} />);
+
+      expect(
+        wrapper
+          .find(".avatar")
+          .childAt(0)
+          .hasClass("inner")
+      ).toEqual(true);
+      expect(wrapper.find("Avatar").props().alt).toEqual(altVariant);
```

Comment:
I don't think we need this. This is that weird test again that is saying "I pass 'sum' the first arugment of "1", now tests did I pass 'sum' the first argument of 1".